### ISSUE TITLE
Does not correctly show wrapped 

### ISSUE THREAD

**Issue by Gamerboi520** [vantezzen/wrapped#3]
It does not worked i put the json file into it and it says I've watched zero videos

![Screenshot_20230603_162744_My Files](https://github.com/vantezzen/wrapped/assets/84089845/91427dbd-b846-4fbc-987d-1be3180cd6d0)

https://github.com/vantezzen/wrapped/assets/84089845/ab58309c-a664-4219-a9a9-f793576b1a18

**Comment by Gamerboi520** [comment]
![image_2023-06-03_163652520](https://github.com/vantezzen/wrapped/assets/84089845/d22b4842-2541-48da-b66b-1c75badc86a0)

**Comment by Gamerboi520** [comment]
Then at the end of the wrapped on PC it only gives data to this information

Total likes | 8,000
Day with most liked posts | 04/23/2023 (x362)
First liked video | 04/16/2023

Which is just completely inaccurate

**Comment by orbboy999** [comment]
I will also confirm that it's incorrect as well. I have the **exact** same issue as this person.

**Comment by hug10101** [comment]
I have this same issue

**Comment by vantezzen** [comment]
Thank you for reporting this - is someone who is having this issue on a PC able to open the JavaScript console and look for/screenshot error that occur after uploading the TikTok file?
It looks like currently a lot of people are experiencing seeing 0 watched videos so TikTok might have changed the export format in some way

**Comment by 4troDev** [comment]
also having same problem

**Comment by vantezzen** [comment]
Yes that would be helpful to find the issue if you can send the file to wrapped@vantezzen.io!

**Comment by Gamerboi520** [comment]
I sent it under gamerboi520@gmail.com just now if you don't see it in inbox check spam folder

**Comment by 4troDev** [comment]
I check the console and this is what it shows [Video On googleDrive](https://drive.google.com/file/d/1XCMKU1ChsZigNkKcmLd4TPR8oQanlE9q/view?usp=sharing)
<img width="1213" alt="Screenshot 2023-06-04 at 10 56 57 AM" src="https://github.com/vantezzen/wrapped/assets/86755192/f26c411c-b46e-4545-82b7-ddcad71f27ec">

**Comment by vantezzen** [comment]
> I sent it under [gamerboi520@gmail.com](mailto:gamerboi520@gmail.com) just now if you don't see it in inbox check spam folder

Thanks I received your file and can reproduce this error - I'll see if I can find the error

**Comment by Gamerboi520** [comment]
> > I sent it under [gamerboi520@gmail.com](mailto:gamerboi520@gmail.com) just now if you don't see it in inbox check spam folder
> 
> Thanks I received your file and can reproduce this error - I'll see if I can find the error

Thanks!

**Comment by vantezzen** [comment]
<img width="196" alt="Screenshot 2023-06-04 at 17 22 06" src="https://github.com/vantezzen/wrapped/assets/10333196/11a8c995-3dba-48b2-8308-aa051468b743">

It looks like the TikTok data watch history is simply actually empty and doesn't contain any information on which videos were watched.
Can I ask which country your TikTok is set to? From what I gathered from comments it seems like the exported data might be different in different countries which would be bad for calculating the statistics.

For now, I'll add a small disclaimer to the page that will get displayed if no watch history can be found.

**Comment by Gamerboi520** [comment]
> <img alt="Screenshot 2023-06-04 at 17 22 06" width="196" src="https://user-images.githubusercontent.com/10333196/243188290-11a8c995-3dba-48b2-8308-aa051468b743.png">
> It looks like the TikTok data watch history is simply actually empty and doesn't contain any information on which videos were watched. Can I ask which country your TikTok is set to? From what I gathered from comments it seems like the exported data might be different in different countries which would be bad for calculating the statistics.
> 
> For now, I'll add a small disclaimer to the page that will get displayed if no watch history can be found.

My tiktok is set to USA which I also did get the USA Government terms of service update thing. and I don't use my VPN when I do watch tiktok. it doesn't make sense for 2 years of watch history data to just not be there. Is there anything I can do or can I just not used wrapped now?

**Comment by Gamerboi520** [comment]
![Screenshot_20230604_114838_TikTok.png](https://github.com/vantezzen/wrapped/assets/84089845/ef294652-078e-4f99-b906-ec1c66c5fa03)

![Screenshot_20230604_114801_TikTok.png](https://github.com/vantezzen/wrapped/assets/84089845/90d4fa98-3530-444a-962e-5ffd5e6f59cc)

Here's what my settings look like

**Comment by vantezzen** [comment]
I've seen some comments from people that don't even see the "Download your data" option in settings so it might be a regional thing.
I'll create a testing TikTok account for the US using a VPN and test if there is some option that allows me to export with watch history. If someone else finds something for that feel free to comment about it here!

**Comment by Gamerboi520** [comment]
Ill retry the data export again without using my VPN at all until it gets finished (Tiktok did take exceptionally long to send me it i requested it on May 22nd and got it Yesterday lol  so much for just 4 days i honestly thought that i just had a lot of data that needed to be process and that's why it was taking so long) i'll keep watch and ill retry when i get the new export bye bye :)

**Comment by Domenico3726** [comment]
> Thanks for the info, I've seen some comments from people that don't even see the "Download your data" option in settings so it might be a regional thing. I'll create a testing TikTok account for the US using a VPN and test if there is some option that allows me to export with watch history. If someone else finds something for that feel free to comment about it here!

For me the error is that if I start the demo it loops in the preparing phase:
![grafik](https://github.com/vantezzen/wrapped/assets/80212883/a19a2a06-427a-4467-9671-fb7868d5745b)

The error appears in page-8f6b54827bf6af8b.js 1:12250
with the error that "Video Browsing History" can't be accessed since e.Activity is undefined:
![grafik](https://github.com/vantezzen/wrapped/assets/80212883/6ed46c64-f8b8-4b52-a02f-1740e075edf0)
![grafik](https://github.com/vantezzen/wrapped/assets/80212883/df249256-a1cc-4b16-862a-7375c106286f)

with more error logs:
![grafik](https://github.com/vantezzen/wrapped/assets/80212883/b2163d92-0706-44e3-93d5-23a75742a4ed)

**Comment by vantezzen** [comment]
Looks like the disclaimer check broke the demo mode - should be fixed again now

**Comment by vantezzen** [comment]
Thanks for the update - I think TikTok's export service might have been overloaded due to the sudden influx in people requesting their data export and so the watch history was excluded.
As for the data not including your whole TikTok history, this is to be expected (see "Does my Wrapped contain my full TikTok history?" in the FAQ on the page). TikTok seems to remove the watch history after some time (or at least doesn't export it anymore).

**Comment by Gamerboi520** [comment]
That makes sense!
